source/js/components/join/join.jsx

```diff
@@ -289,6 +307,29 @@ export default class JoinUs extends React.Component {
     );
   }
 
+  /**
+   * Render localization fields
+   */
+
+  renderLocalizationFields() {
+    let header = this.props.formPosition == `header`;
```

Comment:
`==` is a coercive compare, which I'm pretty sure we don't want?
```suggestion
    let header = this.props.formPosition === `header`;
```